elements/lisk-p2p/src/peer.ts

```diff
@@ -43,7 +43,7 @@ import {
 } from './validation';
 
 // This interface is needed because pingTimeoutDisabled is missing from ClientOptions in socketcluster-client.
-interface ClientOptionsUpdated {
+export interface ClientOptionsUpdated {
```

Comment:
If this is only used in `outbound_peer.ts`, it can be moved to that file and not exported.

Reply:
Same as `SCServerSocketUpdated` type.